traefik/templates/_podtemplate.tpl

```diff
@@ -383,6 +383,33 @@
           - "--tracing.addinternals"
           {{- end }}
 
+          {{- with .Values.tracing }}
+            {{- with .sampleRate }}
+          - "--tracing.sampleRate={{ . }}"
+            {{- end }}
+
+            {{- with .serviceName }}
+          - "--tracing.serviceName={{ . }}"
+            {{- end }}
+
+            {{- range $name, $value := .globalAttributes }}
+          - "--tracing.globalAttributes.{{ $name }}={{ $value }}"
+            {{- end }}
+
+            {{- range $index, $value := .capturedRequestHeaders }}
+          - "--tracing.capturedRequestHeaders[{{ $index }}]={{ $value }}"
+            {{- end }}
+
+            {{- range $index, $value := .capturedResponseHeaders }}
+          - "--tracing.capturedResponseHeaders[{{ $index }}]={{ $value }}"
+            {{- end }}
+
+            {{- if gt (len .safeQueryParams) 0 }}
```

Comment:
```suggestion
            {{- if .safeQueryParams }}
```
An empty array is evaluated as _false_ in helm.
See https://helm.sh/docs/chart_template_guide/control_structures/#ifelse